llvm/test/LTO/AArch64/TestInputs/bar.ll

```diff
@@ -0,0 +1,35 @@
+; This file contains the new semantic of the branch-target-enforcement, sign-return-address.
+; Used for test mixing a mixed link case and also verify the import too in llc.
+
+; RUN: llc %s -o - | FileCheck %s
+
+target datalayout = "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128"
+target triple = "aarch64-unknown-linux-gnu"
+
+define dso_local void @bar() #0 {
+entry:
+  ret void
+}
+; CHECK-LABEL: bar:
```

Comment:
Isn't bar intended to have signed return address since we have `!2 = !{i32 8, !"sign-return-address-all", i32 2}` module flag? The assembly output is just `ret` which looks incorrect. Please let me know if I miss smth

Reply:
`bar` could come from a function that is decorated with `__attribute__((branch-protection=none))` while the file compiled with `-mbranch-protection=bti+pac-ret+b_key`.
Frontend already assigns the function attributes so in this case we want bar to not have any pac instruction.